src/FffGcodeWriter.cpp

```diff
@@ -826,15 +846,21 @@ void FffGcodeWriter::processRaft(const SliceDataStorage& storage)
         constexpr int zag_skip_count = 0;
         constexpr coord_t pocket_size = 0;
 
-        Infill infill_comp(
-            EFillMethod::ZIG_ZAG, zig_zaggify_infill, connect_polygons, raft_outline_path, infill_outline_width, interface_line_spacing,
-            fill_overlap, infill_multiplier, fill_angle, z, extra_infill_shift,
-            interface_max_resolution, interface_max_deviation,
-            wall_line_count, infill_origin, skip_stitching, fill_gaps, connected_zigzags, use_endpieces, skip_some_zags, zag_skip_count, pocket_size
+        for (const auto& raft_outline_path : raft_outline_paths)
```

Comment:
Going by the variable name, I'd think that this would be a variable-width path or a toolpath.

Writing down the type would clarify this. Alternatively, we can consider renaming it to `_centerline`.
```suggestion
        for (const Polygons& raft_outline_path : raft_outline_paths)
```